pyproject.toml

```diff
@@ -0,0 +1,69 @@
+[tool.poetry]
+name = "tds"
+version = "0.1.1"
+description = "The API that sits on top of the data store for the ASKEM program."
+authors = ["Five Grant <five@jataware.com>"]
+readme = "README.md"
+packages = [{include = "src"}, {include="tests"}]
+license = "MIT"
+
+[tool.poetry.urls]
+"Bug Tracker" = "https://github.com/DARPA-ASKEM/data-store-api/issues"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+dbml-builder = "^0.3.0"
+sqlalchemy = "^1.4.42"
+pydantic = "^1.10.2"
+uvicorn = "^0.19.0"
+fastapi = "^0.85.1"
+click = "^8.1.3"
+requests = "^2.28.1"
+psycopg2-binary = "^2.9.4"
+pytest = "^7.2.0"
+pre-commit = "^2.20.0"
```

Comment:
these two should probably be dev dependencies

Reply:
Cool, will fix now

Reply:
fixed